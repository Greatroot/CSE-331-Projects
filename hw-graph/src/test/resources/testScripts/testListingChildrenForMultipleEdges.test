# Tests listing children for a graph with three edges

CreateGraph graph
AddNode graph n1
AddNode graph n2
AddNode graph n3
AddEdge graph n1 n2 e1
AddEdge graph n2 n3 e2
AddEdge graph n1 n3 e3
ListChildren graph n1
ListChildren graph n2
ListChildren graph n3