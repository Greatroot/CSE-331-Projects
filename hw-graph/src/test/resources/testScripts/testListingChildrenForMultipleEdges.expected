# Tests listing children for a graph with three edges

created graph graph
added node n1 to graph
added node n2 to graph
added node n3 to graph
added edge e1 from n1 to n2 in graph
added edge e2 from n2 to n3 in graph
added edge e3 from n1 to n3 in graph
the children of n1 in graph are: n2(e1) n3(e3)
the children of n2 in graph are: n3(e2)
the children of n3 in graph are: